django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -227,6 +230,7 @@
     line-height: 22px;
     margin: 0;
     border-top: 1px solid #ddd;
+    width: 100%;
```

Comment:
The `width: 100%` causes problems for me, it does not play well in combination with margins/paddings. The paginator borders are wider than the changelist itself.